### PR TITLE
Issue#2111 view

### PR DESCRIPTION
**Pull request by Hugo-Hontans**
- [x] Show the view without all the data being loaded

### CODE REVIEW

File: libs/movie/src/lib/movie/components/movie-analytics-chart/movie-analytics-chart.component.ts [hunk 1]
```diff
@@ -97,23 +103,23 @@ export class MovieAnalyticsChartComponent {
...
+      axisBorder: {show: false},
       axisTicks: {show: false}
     };
   }
 
   totalHitsOnCurrentMonth(eventName: MovieAnalyticsEventName) {
-    const total = this.chartData.find(chart => chart.eventName === eventName).y
-    return getSum(total);
+    const total = this.chartData.find(chart => chart.eventName === eventName).y;
+    return total ? getSum(total) : undefined;
```
Comment (Maxstgt) [inline libs/movie/src/lib/movie/components/movie-analytics-chart/movie-analytics-chart.component.ts]: why undefined and not `0`

Comment (Hugo-Hontans) [reply libs/movie/src/lib/movie/components/movie-analytics-chart/movie-analytics-chart.component.ts]: When the analytics is not loaded, we want to display nothing and especially not 0

Comment (GrandSchtroumpf) [reply libs/movie/src/lib/movie/components/movie-analytics-chart/movie-analytics-chart.component.ts]: Same as previous PR. You don't need to specify "return undefined" : 
```typescript
if (total) {
  return getSum(total);
}
```

Comment (Hugo-Hontans) [reply libs/movie/src/lib/movie/components/movie-analytics-chart/movie-analytics-chart.component.ts]: Yes sure !

**Pull request merged**